### ISSUE TITLE
Release zip contains checker.jar that contains test reports

### ISSUE THREAD

**Issue by wmdietl** [typetools/checker-framework#706]
Unzip checker-framework-2.0.0.zip and then do
`unzip -l checker/dist/checker.jar`

The jar file contains unnecessary test report XML files:

```
reports/TEST-tests.CompilerMessagesTest.xml
reports/TEST-tests.FenumSwingTest.xml
reports/TEST-tests.FenumTest.xml
reports/TEST-tests.FormatterTest.xml
```
